LibraryAccess/src/shared/IconTitle.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export const ICONS = {
+  ALIGN_JUSTIFY: 'align-justify',
+  DUMBBELL: 'dumbbell',
+  FINGERPRINT: 'fingerprint',
+  GEM: 'gem',
+  HOME: 'home',
+  ID_CARD: 'id-card',
+  KEY: 'key',
+  LIST_UI: 'list-ui',
+  PAINT_BRUSH: 'paint-brush',
+  SHIELD_ALT: 'shield-alt',
+  STAR: 'star'
+}
+
+
+const IconTitle = ({ faIcon, solid = false, title }) => (
```

Comment:
This is a functional way to make a component, which is useful when you want to make a component that doesn't need state.

Reply:
This icon pattern was replicated a lot of places, so it made sense to me to just make it a reusable component, and just allow for props to change the things that need to be changed